### PR TITLE
4 deliverable 1 problem statement poc plan development plan

### PR DESCRIPTION
**Pull request by yex33**
# Description
Submitting the initial draft for deliverable 1. The documents look good in general.

# Notes
@qstommyshu Can you
- [x] Look at the Technology section in Dev Plan and
see if we need to include more tools for JavaScript/TypeScript development?

### CODE REVIEW

File: docs/Common.tex [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
-\\ Student 2 name
-\\ Student 3 name
-\\ Student 4 name} % AUTHOR NAMES                  
+\newcommand{\progname}{SFWRENG 4G06 Capstone Design Project} % PUT YOUR PROGRAM NAME HERE
+\newcommand{\projname}{MotionMingle} % project name
+\newcommand{\authname}{Team \#18, InfiniView-AI
+\\ Anhao Jiao
+\\ Kehao Huang
+\\ Qianlin Chen
+\\ Shu Qi
```
Comment (KehaoHuang) [inline docs/Common.tex:9]: Switch first name and last name

Comment (yex33) [reply docs/Common.tex:9]: Do we need to put the last name in front? [They](https://github.com/marlon4dashen/Hairesthetics/blob/main/docs/DevelopmentPlan/DevelopmentPlan.pdf) didn't do it.

**Review by KehaoHuang — COMMENTED** [review]
One of the team member's name was not entered correctly in Common.tex.

File: docs/Common.tex (hunk 1 above)
Comment (yex33) [reply docs/Common.tex:9]: Oh it's Shu Qi. What a gamer.

Comment (yex33) [reply docs/Common.tex]: Corrected in baec17f

File: docs/ProblemStatementAndGoals/ProblemStatement.tex [hunk 2]
```diff
@@ -17,40 +17,101 @@
...
 \section{Goals}
 
+\begin{description}
+\item[Learning and teaching experience improvement] The product should provide at
+  least three types of visual effects to show the instructor's motions in Tai
+  Chi lectures, making it easier to see the instructor's motions in video calls.
+\item[Minimizing the hardware requirements for end users] The product should be
+  accessible to users as long as they have a device that supports video
+  conferences. Users should be able to use the product seamlessly as long as
+  they can access the Internet.
```
[the remark this answers is not in the document]
Comment (qstommyshu) [reply docs/ProblemStatementAndGoals/ProblemStatement.tex:96]: Delete the part "User should be able to use the product seamlessly as long as they can access the Interenet"

File: docs/ProblemStatementAndGoals/ProblemStatement.tex [hunk 3]
```diff
@@ -17,40 +17,101 @@
...
+\item[Real-time video post-processing] The speed of video processing and visual
+  effect generation should not be noticeable. There should be no significant lag
+  (< 3 seconds) in the instructor's and practitioner's video calls.
+\item[Safety of User Data] The application should secure user data and minimize the
```
[the remark this answers is not in the document]
Comment (qstommyshu) [reply docs/ProblemStatementAndGoals/ProblemStatement.tex:100]: Change to "The application should secure user data, and no case of data leak should happen through the application."

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 4]
```diff
@@ -17,59 +22,299 @@
...
+
+\subsubsection{Meetings Outside Lectures}
+
+\begin{description}
+\item[Time] Sundays at 8:30 p.m., Wednesdays at 6:00 p.m. or right after the
+  supervisor meeting
+\item[Length] 1 hour
+\item[Frequency] Every week
+\item[Location] Discord
+\item[Attendance Expectation] All team members are present.
```
Comment (jiaoa3) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:91]: All team members must be present.

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 5]
```diff
@@ -17,59 +22,299 @@
...
+In case a team member is unable to attend a meeting, the member must inform the
+team \emph{at least 1 hour before the meeting}. Notice of absence shall be
+reported in the form of comments under the Github issue corresponds to the
+missed meeting. Absence without an ahead-of-time notice counts as a no-show.
+
+No-show in meetings shall be recorded and, as a result, will be reflected in the
+team contribution evaluation factor. Frequent or regular absence of meetings,
+even if the absence is being reported and documented, will also negatively
+impact one member's team contribution factor. For a contribution factor on the
+scale of 0 to 1.1, a tentative conversion from the number of absence or no-shows
```
[the remark this answers is not in the document]
Comment (jiaoa3) [reply docs/DevelopmentPlan/DevelopmentPlan.tex:123]: the contribution factor is range from0 to 1.1

**Review by jiaoa3 — COMMENTED** [review]
few things to address

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 6]
```diff
@@ -17,59 +22,299 @@
...
+
 \section{Team Member Roles}
 
+\begin{table}[hp]
+  \caption{Team member roles} \label{TblTeamMemberRoles}
+  \begin{tabularx}{1.0\linewidth}[h]{smml}
+    \toprule
+    \textbf{Member}            & \textbf{Role}                            & \textbf{Expertise}            & \textbf{Responsibility}                                                                      \\ \hline
+    \midrule
+    Xunzhou (Joe) Ye     & Developer, Co-host Leader          & \LaTeX, Python             & Machine Learning Model, Pipeline Optimizations                                         \\ \hline
```
[the remark this answers is not in the document]
Comment (jiaoa3) [reply docs/DevelopmentPlan/DevelopmentPlan.tex:158]: every two members should be note takers by taking turns

File: docs/DevelopmentPlan/DevelopmentPlan.tex (hunk 4 above)
Comment (yex33) [reply docs/DevelopmentPlan/DevelopmentPlan.tex:91]: Rejected.

**Review by qstommyshu — APPROVED** [review]

**Review by qstommyshu — APPROVED** [review]
GLHF

**Review by jiaoa3 — APPROVED** [review]
All good!

**Pull request merged**